internal/cli/newcontroller/newcontroller.go

```diff
@@ -170,13 +219,15 @@ func (c *Command) views(controller *Controller) (views []*View) {
 }
 
 func (c *Command) view(controller *Controller, action *Action) *View {
+	fmt.Println(action.Name)
```

Comment:
Remove fmt

Reply:
If we wrap it in a mutex, that'd defeat the purpose of having it be in a goroutine. For now, I say we run the scaffolding sequentially instead of concurrently. This isn't exactly a performance critical hot-path anyway and there are lower hanging fruit for us to occupy our time with ;)

Reply:
I removed the concurrent calls and it works as expected